lib/json.c

```diff
@@ -86,3 +86,55 @@ int json_object_object_get_ex(struct json_object *obj, const char *key,
 	return 0;
 }
 #endif
+
+int json_object_add_string(struct json_object *jo, const char *key,
+			   const char *str)
+{
+	struct json_object *jon;
+
+	jon = json_object_new_string(str);
+	if (jon == NULL) {
+		json_object_put(jon);
+		return -1;
+	}
+
+	json_object_object_add(jo, key, jon);
+	return 0;
+}
+
+int json_object_add_bool(struct json_object *jo, const char *key, bool boolean)
+{
+	struct json_object *jon;
+
+	jon = json_object_new_boolean(boolean);
+	if (jon == NULL) {
+		json_object_put(jon);
+		return -1;
+	}
+
+	json_object_object_add(jo, key, jon);
+	return 0;
+}
+
+int json_object_add_int(struct json_object *jo, const char *key, int64_t value)
```

Comment:
What's the difference between json_object_int_add and this function?  Why do we need 2 functions to do the same thing?

Reply:
I removed the imported function in favor to the old one... Same applies for the others.